plugins/ltac/tacintern.ml

```diff
@@ -757,6 +757,26 @@ let glob_tactic_env l env x =
     List.fold_left (fun accu x -> Id.Set.add x accu) Id.Set.empty l in
   intern_pure_tactic { (Genintern.empty_glob_sign ~strict:true env) with ltacvars } x
 
+let intern_strategy ist s =
+  let rec aux stratvars = function
+    | Rewrite.StratVar x ->
+      if Id.Set.mem x.v stratvars then Rewrite.StratVar x.v
```

Comment:
We could make this whole branch `assert false`, since it's unreachable except from plugins.  But maybe it's useful if any plug-in wants to craft a strategy by hand?

Reply:
I would keep it as it, yes. With an explaing comment.

Reply:
Added comment